conf/nomad/nifi.hcl

```diff
@@ -8,8 +8,8 @@ job "${service_name}" {
     max_parallel      = 1
     health_check      = "checks"
     min_healthy_time  = "10s"
-    healthy_deadline  = "12m"
-    progress_deadline = "15m"
+    healthy_deadline  = "35m"
+    progress_deadline = "40m"
```

Comment:
I suggest that we keep the old values here, since this did not help with our problem

Reply:
okei

Reply:
done